src/bundle-manifest.ts

```diff
@@ -201,16 +201,21 @@ export function generateSharedDepsMergeStrategy(minEntrypoints?: number):
  * bundle for an application shell.
  */
 export function generateShellMergeStrategy(
-    shell: UrlString, minEntrypoints: number): BundleStrategy {
+    shell: UrlString, minEntrypoints?: number): BundleStrategy {
   return (bundles: Bundle[]): Bundle[] => {
+    if (minEntrypoints === undefined) {
```

Comment:
+1 one on this change, but agree a test would be good. Feel free to revert this for now as unrelated and address in separate PR.